Gems/ROS2/Code/Include/ROS2/Manipulation/Controllers/JointsPositionControllerRequests.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/string/string.h>
+#include <ROS2/Manipulation/JointsManipulationRequests.h>
```

Comment:
It doesn't look like all the includes are needed/used and can likely be removed
```suggestion
#include <AzCore/std/string/string.h>
```

Reply:
JointsManipulationRequest.h will be replaced with JointInfo.h (a smaller include which is needed). Otherwise, will apply, nice catch!